1904-12-15.xml

```diff
@@ -0,0 +1,2146 @@
+                        by the Sakelaridis family, who charned lovers of music in Munich and Athens
+                        last year. The Alexandrian public, for the | first time, will have the
+                        opportunity of hearing popular Greek songs which are considered by the
+                        highest musical authorities of Europe as the pure inheritance of ancient
+                        Greek music.</p>
+                </div>
+                <div type="item">
+                    <head>COTTON SPINNING PROFITS.</head>
+                    <p>Mr. William Tattersall, of Manchester, in his private circular on Friday
+                        last, says that the profitable state of spinning will hardly show itself
+                        this year. The profit for the past twelve months, Lending November 30, for
+                        about eighty-seven concerns, is only £30,000. Last year for ninety mills the
+                        loss was about $44,000. The capital employed in these mills is upwards of
+                        £3,000,000.</p>
+                </div>
+                <div type="item">
+                    <head>IS IT MURDER!</head>
+                    <p>The dead body of a native man was found by the Alexandria city police
+                        yesterday evening near the Mahmoudieh Canal. It was conveyed to the
+                        Government Hospital, where the medical examination showed that la rib had
+                        been broken from the blow of a nabout. The brothers of the divorced wife of
+                        the deceased were arrested on suspicion of being the murderers, as they
+                        swore to kill him on account of their sister, his late wife,</p>
+                </div>
+            </div>
+            <cb n="3"/>
+            <div type="section">
+                <head>THE KHEDIVE.</head>
+                <p>The Montazah Mosque has been decorated in honour of the anniversary of the
+                    birthday of the elder son of his Highness the Khedive, who enters on his eighth
+                    year to-morrow. His Highness will attend Friday prayers at that mosque. In the
+                    evening &amp; large number of Uleina and Egyptian notables and officials will be
+                    invited at Montazah, where prayers will be given and the Korau read.</p>
+                <p>In honour of the birthday of the Prince, the | Mosque of Abul Abbas Mursy at
+                    Alexandria has been decorated, and this evening at 8 p.m. prayers will be given
+                    which will be followed | by reading from the Koran-el-Cheief.</p>
+                <p>The Khedive will probably return to Cairo from Montazh on Saturday afternoon.</p>
+                <p>His Highness will preside at the Council of Ministers which is expected to meet
+                    on Monday.</p>
+                <div type="item">
```

Comment:
add a `</div>`--the Khedive div is its own item, and what follows is a sibling item